Revit_oM/Enums/Category.cs

```diff
@@ -22,715 +22,1299 @@
 
 using BH.oM.Base.Attributes;
 using System.ComponentModel;
+using BH.oM.Revit.Attributes;
 
 namespace BH.oM.Revit.Enums
 {
     [Description("A collection of Revit categories supported by BHoM (UI, filtering etc.).")]
     public enum Category
     {
         [DisplayText("Abutment Foundation Tags")]
+        [CategoryType("Annotation")]
+        [CategoryDiscipline("Infrastructure")]
         OST_AbutmentFoundationTags,
         [DisplayText("Abutment Pile Tags")]
+        [CategoryType("Annotation")]
+        [CategoryDiscipline("Infrastructure")]
         OST_AbutmentPileTags,
         [DisplayText("Abutment Tags")]
+        [CategoryType("Annotation")]
+        [CategoryDiscipline("Infrastructure")]
         OST_BridgeAbutmentTags,
         [DisplayText("Abutment Wall Tags")]
+        [CategoryType("Annotation")]
+        [CategoryDiscipline("Infrastructure")]
         OST_AbutmentWallTags,
         [DisplayText("Abutments")]
+        [CategoryType("Model")]
+        [CategoryDiscipline("Infrastructure")]
         OST_BridgeAbutments,
         [DisplayText("Adaptive Points")]
+        [CategoryType("Annotation")]
+        [CategoryDiscipline("Architecture", "Structure", "Mechanical", "Electrical", "Piping")]
         OST_AdaptivePoints,
-        [DisplayText("Air Systems")]
```

Comment:
Curious, Why are some categories left out of the list?

Reply:
Because now I took only the categories that reside in `Document.Settings.Categories` and their `CategoryType != CategoryType.Internal`, i.e. the list should reflect what you can find in Visibility & Graphics window in Revit.

A few internal ones (Assemblies, Groups, Revisions, Schedules, Views) were added as they are expected to be relevant to our workflows. Ofc we can freely extend the list, but for now I kept it as small as possible to avoid congestion in the UI.